egs2/chime7_task1/asr1/local/gen_task1_data.sh

```diff
@@ -51,4 +51,13 @@ fi
 if [ ${stage} -le 2 ] && ! contains $skip_stages 2; then
   python local/gen_task1_data.py -c $chime6_root \
       -d $dipco_root -m $mixer6_root -o $chime7_root --eval_opt $gen_eval
+  if [ $gen_eval -le 0 ]; then
```

Comment:
It would be good to add a comment on how the generated ID looks like if we use a bit complicated bash commands.

Reply:
Added, also many thanks your comment reminded to also add UEM files in the JSON MD5 digest check script. 
This script https://github.com/espnet/espnet/blob/7da60716fb41c04a97923a84b4babb4dae461fa1/egs2/chime7_task1/asr1/local/check_data_gen.py checks for data integrity and if participants have generated successfully the whole dataset.